cfn-resources/cloud-backup-schedule/cmd/resource/resource.go

```diff
@@ -147,6 +147,17 @@ func cloudBackupScheduleCreateOrUpdate(req handler.Request, prevModel *Model, cu
 	}
 
 	params := currentModel.getParams()
+	// From https://jira.mongodb.org/browse/HELP-55421
+	// Even after deleting and recreating the schedules, it is required to fetch the ID of the policy in order to be passed to the update request.
+	if len(params.Policies) == 1 && params.Policies[0].GetId() == "" {
+		backupSchedule, resp, err := client.AtlasV2.CloudBackupsApi.GetBackupSchedule(context.Background(), *currentModel.ProjectId, *currentModel.ClusterName).Execute()
```

Comment:
```suggestion
		backupSchedule, resp, err := client.Atlas20231115002.CloudBackupsApi.GetBackupSchedule(context.Background(), *currentModel.ProjectId, *currentModel.ClusterName).Execute()
```
`AtlasV2` needs to be changed to `Atlas20231115002` because this [PR](https://github.com/mongodb/mongodbatlas-cloudformation-resources/pull/919) has been merged